sqlparser/sql.y

```diff
@@ -3010,6 +3038,10 @@ set_list:
   {
     $$ = append($1, $3)
   }
+| sql_id TO value_expression ',' value_expression
+  {
+    $$ = SetExprs{&SetExpr{Name: $1, Expr: $3}, &SetExpr{Name: $1, Expr: $5}}
```

Comment:
hm... how it works with `SET name TO 1,2,3,4`? or `SET name to 'str1', 'str2', 'str3'`? for amount of values > 2
I saw that such cases processed with `append` items to the list, like `$$ = append($$, ...)`

Reply:
Yes, you are right, in this case we need probably introduce a new type like `set_to_list` or something similar to handle such thing.